arbitrum-docs/partials/_glossary-partial.md

```diff
@@ -57,7 +57,7 @@
 ### Blockchain {#blockchain}
 <p>A distributed digital ledger that is used to record transactions and store data in a secure, transparent, and tamper-resistant way, notably in cryptocurrency protocols. </p>
 
-### 🧢 CAP Finance {#cap-finance}
```

Comment:
It looks like your notion-docs package is not up to date (it shouldn't remove these icons). Can you run `yarn install` inside that folder and re-run `upgrade:all`?